az-digital/az_quickstart
# Closes #2017 Add Landing Page functionality to Quickstart

## Description
Adds landing page functionality to Quickstart in the form of Marketing Campaign Pages. By selecting a Marketing Campaign Page style for a page, the page will be displayed without the navigation menu and other page regions to focus the user on a single call-to-action.

Please see [the associated issue](https://github.com/az-digital/az_quickstart/issues/2017) for the full requirements for this functionality. New documentation about this feature will be added to the [Adding Pages](https://quickstart.arizona.edu/pages) page on the Quickstart site.

## Related issues




#2017 

## How to test



1. View the Marketing Campaign Page example in the demo content.
2. Edit a page or create a new page to display with a Marketing Campaign Page style.
3. Test different Marketing Campaign Page styles.
4. Confirm that Content Admins can select Marketing Campaign Page styles for pages and Content Editors cannot.
6. Test disabling/enabling Marketing Campaign Page styles site-wide in Quickstart Settings.

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
